src/ray/common/ray_config_def.h

```diff
@@ -1001,3 +1001,12 @@ RAY_CONFIG(bool, start_python_gc_manager_thread, true)
 // Whether to enable the feature of outputting error log if the task is
 // still retryable.
 RAY_CONFIG(bool, enable_output_error_log_if_still_retry, true)
+
+// Whether to enable GCS ray_syncer sync messages batching.
+RAY_CONFIG(bool, gcs_ray_syncer_batching_enabled, false)
+
+// The batch size for ray_syncer_bidi_reactor to sync messages to other nodes.
+RAY_CONFIG(size_t, syncer_batch_size, 1)
```

Comment:
Trying to wrap my head around this one.  Does 1 mean that batching is disabled (as in, it'll only send one message in a batch) or does 1 mean that's it's super enabled and will trying to compress as many messages as possible into a larger outer message?